modules/apps/notification/notification-rest-test/src/testIntegration/java/com/liferay/notification/rest/resource/v1_0/test/NotificationQueueEntryResourceTest.java

```diff
@@ -172,11 +201,69 @@ public void testDeleteNotificationQueueEntryBatch() throws Exception {
 		super.testDeleteNotificationQueueEntryBatch();
 	}
 
-	@Ignore
 	@Override
 	@Test
 	public void testGetNotificationQueueEntriesPage() throws Exception {
-		super.testGetNotificationQueueEntriesPage();
```

Comment:
@fabioafreitas why did you need to remove the `super.testGetNotificationQueueEntriesPage();`?